memgpt/agent_store/local.py

```diff
@@ -252,9 +252,11 @@ def json_to_message(self, message_json) -> Message:
             user_id=self.config.anon_clientid,
             agent_id=self.agent_config.name,
             role=message["role"],
+            name=message["name"] if "name" in message else None,
             text=message["content"],
             model=self.agent_config.model,
             created_at=parse_formatted_time(timestamp),
+            # TODO(swooders) this should be tools, right?
```

Comment:
@sarahwooders these should be revised to the tool-call schema, right?

Reply:
removed this file